.github/workflows/release-please.yml

```
@@ -27,10 +27,15 @@ permissions:
   release-please:
     runs-on: ubuntu-latest
+    # Note: per-package `skip-github-release: true` in release-please-config.json
+    # means release-please-action never emits the `<path>--tag_name` outputs that
+    # would otherwise gate the publish jobs. Instead, detect a release-PR merge by
+    # matching the canonical commit subject produced by release-please:
+    #   chore(main): release <component> <version>
     outputs:
-      npm_released: ${{ steps.release.outputs['packages/npm--tag_name'] }}
-      pypi_released: ${{ steps.release.outputs['packages/pypi--tag_name'] }}
-      dataform_graph_cli_released: ${{ steps.release.outputs['packages/dataform-graph-cli--tag_name'] }}
+      npm_released: ${{ contains(github.event.head_commit.message, 'chore(main): release npm-package') }}
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)